plugins/generator-1.21.1/neoforge-1.21.1/procedures/utils/block_inventory/get_item.java.ftl

```diff
@@ -0,0 +1,8 @@
+private static ItemStack itemFromBlockInventory(LevelAccessor world, BlockPos pos, int slot) {
+	if (world instanceof ILevelExtension ext) {
+		IItemHandler itemHandler = ext.getCapability(Capabilities.ItemHandler.BLOCK, pos, null);
+		if (itemHandler != null)
+			return itemHandler.getStackInSlot(slot).copy();

```

Comment:
Count procedure does not need copy. Do the copy in the procedure code for "item_inslot" directly, not here

Reply:
:heavy_check_mark: Fixed